osd/aws/ROSA_AWS_invalid_permissions.md

```diff
@@ -0,0 +1,8 @@
+{
+    "severity": "Error",
+    "service_name": "SREManualAction",
+    "cluster_uuid": "${CLUSTER_UUID}",
+    "summary": "Installation blocked, action required",
+    "description": "cluster's installation is blocked due to missing or insufficient privileges on the AWS account used to provision the cluster. Please review and validate the pre-requisites required on your AWS account for the installation to succeed. Pre-requisites are described in this document - https://docs.openshift.com/rosa/rosa_getting_started/rosa-aws-prereqs.html",
```

Comment:
nit: we removed the "Your " from the email template so you'll need to make this work as a standalone sentence. Like "Your cluster's installation..."

Reply:
Thanks, I've addressed this now. I see other templates that have a similar issue, so I'll submit a separate PR to fix them.